ci/build.sh

```diff
@@ -139,6 +139,7 @@ RUST_NIGHTLY_LINUX_TARGETS="\
 aarch64-fuchsia \
 armv5te-unknown-linux-gnueabi \
 armv5te-unknown-linux-musleabi \
+armv7-unknown-linux-uclibceabihf \
```

Comment:
Is this the right way to enable CI testing for this target?  I couldn't really untangle how this was meant to work.

Reply:
This actually needs to go into `RUST_LINUX_NO_CORE_TARGETS` instead because the uclibc targets are not distributed via rustup and need to be built with `-Zbuild-std`.

Reply:
Fixed.  I added a comment to help the next person.  I tried to test this via `ci/run-docker.sh armv7-unknown-linux-uclibceabihf`

But I get the following error: `ci/run.sh: 112: cargo: not found`

I suspect there's something messed up with my setup because the same issue occurs for `x86_64-unknown-linux-gnu`.

Reply:
Oh I found it - i had a toolchain override in place to my stage2 custom build which did not include cargo.